xarray/core/npcompat.py

```diff
@@ -1,247 +0,0 @@
-# Copyright (c) 2005-2011, NumPy Developers.
-# All rights reserved.
-
-# Redistribution and use in source and binary forms, with or without
```

Comment:
What do you think of leaving an empty file with the copyright notice.

It'll create a lot of conflicts if there's an open PR touching this file (and its not uncommon to have to add some compat code here)